src/features/device-state/routes/state.ts

```diff
@@ -301,28 +367,83 @@ export const stateV2: RequestHandler = async (req, res) => {
 	const { apiKey } = req;
 	events.emit('get-state', uuid, { apiKey });
 
+	try {
+		const device = await getDevice(req, uuid);
+		const config = getConfig(device) ?? {};
+
+		const appsForState: Dictionary<AppV2> = {};
+
+		const userApp = getUserAppForState(device, config);
+		if (userApp) {
+			const userAppV2 = {
+				commit: userApp.commit,
+				name: userApp.name,
+				releaseId: userApp.releaseId,
+				networks: userApp.networks,
+				services: userApp.services,
+				volumes: userApp.volumes,
+			} as AppV2;
```

Comment:
```suggestion 
			const userAppV2: AppV2 = {
  				commit: userApp.commit,
  				name: userApp.name,
  				releaseId: userApp.releaseId,
  				networks: userApp.networks,
  				services: userApp.services,
  				volumes: userApp.volumes,
  			};
```
Typing it like this ensures the value you assign *is* the type you expect, whereas the `as` form coerces the compiler to think that whatever value you assigned is that type, which is not useful here.

Reply:
Changes made.. resolving since Akis is out.